src/detect-engine-register.h

```diff
@@ -54,7 +54,18 @@ enum DetectKeywordId {
     DETECT_FLOW,
     /* end prefilter sort */
 
+    /* values used in util-var.c go here, to avoid int overflows
+     * TODO update var logic to use a larger type. */
```

Comment:
is this `todo` intended to be left here? a Redmine ticket seems more appropriate to me